components/layout/index--infinite.jade

```diff
@@ -0,0 +1,41 @@
+- colorClass = sd.IS_INVERTED ? 'is-inverted' : ''
+- loggedInClass = user ? 'is-logged-in' : 'not-logged-in'
+- extraClasses = ''
```

Comment:
I can't really refactor these out for now but whenever you include *any* variables inline within a template, ensure they are at the very top.